test/index.test.js

```diff
@@ -39,6 +39,15 @@ describe('Protractor Flake', () => {
   })
 
   context('failed specs', () => {
+    it('calls callback with an err if a bad protractorPath is specified', (done) => {
```

Comment:
I think you can leave this test out since it just asserts on status and we've already tested the case where it uses the custom protractor path